src/ecrc-frontend/src/components/page/transition/Transition.stories.js

```diff
@@ -11,9 +11,17 @@ const header = {
   name: "header name"
 };
 
-export const Default = () => <Transition header={header} />;
+const page = {
+  header
+};
+
+export const Default = () => <Transition page={page} />;
+
+export const NotWhitelisted = () => (
```

Comment:
should we add a story for bcsc case here?

Reply:
BCSC is currently the default case so this page can be used as a redirect point on failure/cancel login at bcsc. Can rename Default to indicate this.

Reply:
yes pls, thanks :)